.github/workflows/ci.yml

```diff
@@ -132,6 +162,11 @@ jobs:
         with:
           python-version: 3.9
 
+      - name: Install dependencies for Qt 5 on Ubuntu
```

Comment:
Is it also needed to install Qt 5 "just" to build the doc ?

Reply:
Yes otherwise I faced an issue with imports of nectarchain. I think because now pyqt is explicitly imported on top of one submodule